cristin-person/src/main/java/no/unit/nva/cristin/person/employment/query/QueryPersonEmploymentHandler.java

```diff
@@ -44,7 +44,6 @@ public QueryPersonEmploymentHandler(QueryPersonEmploymentClient apiClient, Envir
     protected SearchResponse<Employment> processInput(Void input, RequestInfo requestInfo, Context context)
         throws ApiGatewayException {
 
-        AccessUtils.validateIdentificationNumberAccess(requestInfo);
         logger.info(LOG_IDENTIFIERS, extractCristinIdentifier(requestInfo), extractOrgIdentifier(requestInfo));
```

Comment:
In another handler, you have moved logging to  validateRequest.
Done inside a common validate routine, it would be easier to maintain and handling would also be identical.

Reply:
So move all common logic into one method/class?

Reply:
It was more of an example of what can easily happen when code is implemented in the handler.
with regards to my earlier comments.